packages/discovery-provider/src/queries/get_tracks.py

```diff
@@ -31,7 +31,7 @@ class RouteArgs(TypedDict):
     slug: str
 
 
-class GetTrackArgs(TypedDict):
+class GetTrackArgs(TypedDict, total=False):
```

Comment:
Is this used?

Reply:
yes, this fixes the type so that not all the args are required

Reply:
https://mypy.readthedocs.io/en/stable/typed_dict.html#totality